.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Patch for Python 3.6
+      if: matrix.python-version == 3.6
+      run: |
+        pip install strip-hints==0.1.9
+        ./patch-python3.6.sh
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt
+
+    - name: Install dev dependencies
+      run: |
+        pip install -r dev_requirements.txt
+
+    - name: Test
+      run: |
+        pytest
+
+    - name: Build package
+      run: |
+        PYTHON_VERSION=${{ matrix.python-version }}
```

Comment:
Not sure if better but people seem to be using the env syntax:

    - name: Build package
      env:
        PYTHON_VERSION: ${{ matrix.python-version }}
      run: |
        PYTHON_VERSION=${{ matrix.python-version }}

Reply:
forgot about env: much better :)